### DISCUSSION TITLE
⏳ 🔗 integrate Cloudflare’s Turnstile into Elementor forms

### DISCUSSION THREAD

**Discussion by douvan** [elementor/elementor#21151] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Elementor currently integrates Google reCAPTCHA in the form widget. Cloudflare Turnstile is an alternative to reCAPTCHA. Turnstile integration with Elementor forms brings frustration-free, CAPTCHA-free web experiences to website visitors.
By integrating Turnstile into the Elementor form widget, Turnstile stops abuse and confirms visitors are real without the data privacy concerns or awful UX that CAPTCHAs thrust on users.

### Describe the solution you'd like

Allow the Elementor form widget to accept reCAPTCHA in addition to Turnstile as an option for the designer to choose which bot protection service to use on the form.

### Describe alternatives you've considered

Third-party plugin
Simple Cloudflare Turnstile
The new user-friendly alternative to CAPTCHA
By Elliot Sowersby, RelyWP
https://wordpress.org/plugins/simple-cloudflare-turnstile/

**Comment by costrut** [comment]
I would also love to see support for Turnstile built-in to elementor!

Google recaptcha services are really bad UX wise, we have lots of clients on our site who abandon carts or account creation processes because of google's recaptcha
